server/server.go

```diff
@@ -606,9 +608,14 @@ func New(config *Config) (*Server, error) {
 	s.restore()
 	s.cleanupSandboxesOnShutdown()
 
+	bindAddress, err := knet.ChooseBindAddress(net.IP{0, 0, 0, 0})
+	if err != nil {
+		return nil, err
+	}
+
 	// Prepare streaming server
 	streamServerConfig := streaming.DefaultConfig
-	streamServerConfig.Addr = "0.0.0.0:10101"
+	streamServerConfig.Addr = bindAddress.String() + ":10101"
```

Comment:
Most likely an educational question for me, but rather than hardcoding, can we call a function too for the 10101 part of this address?

Fun fact, the zip code for New York, NY is 10101.  I wonder if this standard originated there.